krate/src/main/kotlin/hu/autsoft/krate/validated/optional/ValidatedStringDelegate.kt

```diff
@@ -0,0 +1,22 @@
+package hu.autsoft.krate.validated.optional
+
+import hu.autsoft.krate.Krate
+import hu.autsoft.krate.validated.exception.KrateValidationException
+import hu.autsoft.krate.optional.StringDelegate
+import kotlin.properties.ReadWriteProperty
+import kotlin.reflect.KProperty
+
+internal class ValidatedStringDelegate(
```

Comment:
I've added an implementation that satisfies not using default lambdas. 

We use class delegation here because `StringDelegate` is not `open` for extension.